examples/model_compress/main.py

```diff
@@ -0,0 +1,291 @@
+import os
+import argparse
```

Comment:
Perhaps use another PR to submit examples if they are not specific to gradient pruners?